ga4gh/backends.py

```diff
@@ -365,18 +370,24 @@ class Backend(object):
     """
     Superclass of GA4GH protocol backends.
     """
-    def __init__(self, dataDir, Dataset):
+
+    def __init__(self, dataDir, variantSet):
```

Comment:
Done.
Note: There is no VariantSet superclass. VariantSets just inherit from Object. Here we expect only an object that has a constructor and some methods. A later PR should create a VariantSet superclass for sure.